apps/hash-ai-worker-ts/src/activities/infer-entities/persist-entities.ts

```diff
@@ -221,12 +221,10 @@ export const persistEntities = async (params: {
     });
   };
 
-  const createInferredEntityNotification = async ({
+  const createInferredEntityUpdateNotification = async ({
```

Comment:
I thought what we wanted to do here was to retain notifications for _non-draft_ actions (since they won't appear in the drafts queue) – i.e. we check if `createAs === "live"`, and send the notification if true. Rather than disabling notifications on creation (entities can be created as non-draft).

Reply:
Apologies, you're right - have addressed in https://github.com/hashintel/hash/pull/4182/commits/1dcd06dc661426c0821fd1a1aa57f869d8ca6cbb. Now there'll be notifications for draft/non-draft updates, and non-draft creations in the graph

Reply:
I don't think we want notifications for draft updates either - they will also be actions (approve or reject the update)

Reply:
Makes sense, have done so in https://github.com/hashintel/hash/pull/4182/commits/dc641c78a04df8e021c93b368215e53d23934666